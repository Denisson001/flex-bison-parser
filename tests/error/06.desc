Runtime error - assignment func: negative index