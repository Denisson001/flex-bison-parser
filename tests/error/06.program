{
    string_s = "0123456789";
    string_s[42 - 100] = "8";
}