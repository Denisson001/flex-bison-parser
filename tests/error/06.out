Interpreter: runtime error - index assignment: invalid index
